### ISSUE TITLE
Vista styled start menu

### ISSUE THREAD

**Issue by Zeldaboy14** [Open-Shell/Open-Shell-Menu#1079]
Basically, a vista-esc style added to open shell. Before the 7 peeps say "they are the same":
Windows Vista's start menu had a white gradient that went up and 7's didn't do this
Plus the shutdown option was a power symbol with no words, and a second button could be configured next to it (iirc that is)

Maybe just added a vista toggle to the 7 skin if making vistas separate doesn't work out

The images i provided show the difference between vistas start menu and 7's
![start](https://user-images.githubusercontent.com/55566323/180319514-77427922-079a-4d6c-add9-0b3a7a97c87c.png)
![sshot20091028222007](https://user-images.githubusercontent.com/55566323/180319521-b8b4b998-0e96-4a73-8965-a8fb6cd70558.png)

**Comment by blackcrack** [comment]
this project lives by  participation, if you be in able to programming.. 
or search for a programmer who be in able to do this.. 
would nice if we have more programmers here in this project..
best
Blacky

**Comment by DellZHackintosh** [comment]
There is [an old skin](http://www.classicshell.net/forum/viewtopic.php?f=17&t=5655).Is that useful?

**Comment by Zeldaboy14** [comment]
No, because there's in-accuracies with that skin (the shutdown buttom especially)
